recursion/recursion.md

```diff
@@ -7,11 +7,10 @@ About 4.5 hours
 - 240 minutes for Independent Practice & Challenge
 
 ### Prerequisites
-- JS 1 lesson
-- JS 2 lesson
-- JS 3 lesson
-- JS 4 lesson
-- Score of __15__ or higher at Coding Challege Assesment, Level 1
+- (JavaScript 1 lesson)[https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-1.md]
```

Comment:
Please use relative paths. Example, replace:
`https://github.com/Techtonica/curriculum/blob/master/html/html.md`
With:
`../html/html.md`

Reply:
This is wrong syntax too! 😔 Swap the `(text)[https://path/]` to `[text](https://path/)`, use an editor like Typora.io to get these sorted. 😇